dsn_test.go

```diff
@@ -62,137 +147,15 @@ func TestDsnDeserializeInvalidJSON(t *testing.T) {
 	}
 }
 
-func TestValidDsnInsecure(t *testing.T) {
```

Comment:
Now covered by "MinimalInsecure" test case.
https://github.com/getsentry/sentry-go/pull/112/files#diff-eeb2e1d2b086ffdc5e8a6e60178702aaR44

Except for the custom port, tested in "AllFields".